
syntax = "proto3";
package prophet.proto;


import 'ProphetModel.proto';

option py_generic_services  = true;

service ProphetService
{
  rpc forecast(ProphetRequest) returns(Response);
}